eclair-core/src/main/scala/fr/acinq/eclair/channel/fsm/ChannelOpenDualFunded.scala

```diff
@@ -391,7 +525,19 @@ trait ChannelOpenDualFunded extends DualFundingHandlers with ErrorHandlers {
           val realScidStatus = RealScidStatus.Temporary(RealShortChannelId(blockHeight, txIndex, commitments.commitInput.outPoint.index.toInt))
           val (shortIds, channelReady) = acceptFundingTx(commitments, realScidStatus = realScidStatus)
           d.deferred.foreach(self ! _)
-          goto(WAIT_FOR_DUAL_FUNDING_READY) using DATA_WAIT_FOR_DUAL_FUNDING_READY(commitments, shortIds, channelReady) storing() sending channelReady
+          val otherFundingTxs = allFundingTxs.filter(_.commitments.commitInput.outPoint.txid != confirmedTx.txid).map(_.fundingTx)
+          rollbackDualFundingTxs(otherFundingTxs)
```

Comment:
Did we miss this `rollbackDualFundingTxs` in #2390?

Reply:
Right, but it really doesn't matter as we simply couldn't have multiple funding txs since RBF wasn't implemented.

Reply:
I think we should add a `val fundingTxid = this.commitInput.outPoint.txid` to `Commitments`, it would be more readable and  it's used 36 times (!) in the source code.

Reply:
Definitely, this was long overdue! Done in https://github.com/ACINQ/eclair/pull/2275/commits/135870bfb2f1660d74ab3eac7a07475b121651bb